File: C#/C#preliminary/05_面向对象/04Structure/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _04Structure
{
    class Student
    {
        private string _name;
        public string Name
        {
            get { return _name; }
            set { _name = value; }
        }
        private int _age;
        public int Age
        {
            get { return _age; }
            set
            {
                if (value < 0 || value > 100)
                    value = 0;
                _age = value;
            }
        }
        private char _gender;
        public char Gender
        {
            get
            {
                if (_gender != '男' && _gender != '女')
                    return _gender = '男';
                return _gender;
            }
            set { _gender = value; }
        }

        public Student()
        {

        }
    }
}

File: Java/javaSE/基础语法/src/com/Oop/Static/Student.java
package com.Oop.Static;

public class Student {
    private static int age = 13;
    private double score = 99.5;

    public static void run(){
        System.out.println("run");
    }
    public void go(){
        System.out.println("go");
    }

    public static void main(String[] args) {
        Student s1 = new Student();
        System.out.println(Student.age);
        System.out.println(s1.age);
//        System.out.println(Student.score);
        System.out.println(s1.score);
        Student.run();
        s1.run();
//        Student.go();
        s1.go();
    }
}
